db.js

```diff
@@ -0,0 +1,68 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost:27017/overview', { useNewUrlParser: true, useUnifiedTopology: true });
+
+const { Schema } = mongoose;
+
+const FeatureSchema = new Schema({
+  feature: String,
+  value: String,
+});
+
+const PhotoSchema = new Schema({
+  thumbnail_url: String,
+  url: String,
+});
+
+const StyleSchema = new Schema({
+  style_id: Number,
+  name: String,
+  original_price: String,
+  sale_price: String,
+  'default?': Number,
```

Comment:
Why not just make this default without the '?'. All this needs to hold is a boolean or a number (0 or 1). As long as you know it means default?, you dont need to actually set it like that.